libmscore/tempotext.cpp

```diff
@@ -164,6 +165,26 @@ QString TempoText::duration2tempoTextString(const TDuration dur)
       }
 
 //---------------------------------------------------------
+// updateScore
+//---------------------------------------------------------
+void TempoText::updateScore()
+      {
+      if (segment())
+            score()->setTempo(segment(), _tempo);
+      score()->fixTicks();
+      score()->setPlaylistDirty();
+      }
+
+//---------------------------------------------------------
+// updateRelative
+//---------------------------------------------------------
+void TempoText::updateRelative()
+      {
+      qreal tempoBefore = score()->tempo(this->tick()-1);
```

Comment:
drop the "this" and put space around -

qreal tempoBefore = score()->tempo(tick() - 1);